sdcm/prometheus.py

```diff
@@ -103,3 +106,103 @@ def event_stop(self, disrupt):
             self._disrupt_gauge.labels(disrupt).dec()  # pylint: disable=no-member
         except Exception as ex:  # pylint: disable=broad-except
             LOGGER.exception('Cannot stop metrics event: %s', ex)
+
+
+class PrometheusAlertManagerListener:
+    interval = 10
+
+    def __init__(self, ip, port=9093):
+        self._alert_manager_url = f"http://{ip}:{port}/api/v2"
+
+    def start(self):
+        self.to_stop_flag = threading.Event()
```

Comment:
rule: never define instance attributes outside the constructor. (User of the class should understand what are the instance attributes from the constructor and not go over all the method definitions)

Reply:
fixed.